src/System.Security.Cryptography.X509Certificates/tests/X509FilesystemTests.Unix.cs

```diff
@@ -90,7 +94,7 @@ public static void X509Store_OpenExisting_Fails()
         }
 
         [Fact]
-        private static void X509Store_AddReadOnly()
```

Comment:
Was xunit running these?  I never remember whether it runs private test methods.

Reply:
It seems to.  I went to a recent PR, jumped to their "Test Results" tab on AzDo and changed the filters to be the X509Certificates.Tests library, cleared the results filter and put `X509FilesystemTests` in the name filter, and this one shows up.

This just removes the need to remember/experiment :)